tests/Language/PureScript/Ide/SourceFile/IntegrationSpec.hs

```diff
@@ -31,6 +31,11 @@ spec = beforeAll_ setup $
 
 testCase :: Text -> (Int, Int) -> IO ()
 testCase s (x, y) = do
-  P.SourceSpan f (P.SourcePos l c) _ : _ <- Integration.getInfo s
+  P.SourceSpan f (P.SourcePos l c) _ <- getLocation s
   toS f `shouldSatisfy` T.isSuffixOf "SourceFileSpec.purs"
   (l, c) `shouldBe` (x, y)
+
+getLocation :: Text -> IO P.SourceSpan
+getLocation s = do
+  (_, _, _, Just location) : _ <- Integration.getType s
```

Comment:
Sure, but in that case I want the test to fail anyway. The pattern match failure should point me at this line and I'll know what's up.